lib/routes/jiqizhixin/index.ts

```diff
@@ -0,0 +1,46 @@
+import ofetch from '@/utils/ofetch';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+import { Route, DataItem } from '@/types';
+
+export const route: Route = {
+    path: '/index',
+    name: '机器之心首页',
+    url: 'jiqizhixin.com',
+    maintainers: ['ovo-tim'],
+    example: '/jiqizhixin/index',
+    description: '通过请求 HTML 获取机器之心首页，可获取10条',
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    handler: async (_) => {
+        const baseUrl = 'https://www.jiqizhixin.com';
+        const response = await ofetch(baseUrl, {
+            headers: {
+                'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.110 Safari/537.3',
```

Comment:
RSSHub does have a user agent of [Chrome of macOS](https://github.com/DIYgod/RSSHub/blob/26993f9ebfd917ee42a1c1574b316c614abed2bd/lib/config.ts#L380). Are there any specific reasons to use this fixed string of old Chrome on Windows?

Reply:
Fixed.